### ISSUE TITLE
SDK para Xamarin

### ISSUE THREAD

**Issue by balivo** [stone-payments/sdk-android#22]
Há previsão de implementação de uma biblioteca para utilizar no Xamarin.Android ou Xamarin.iOS?

**Comment by filpgame** [comment]
Olá @balivo,
Cara, por enquanto não temos previsão para criação de uma biblioteca em Xamarin. Mas estamos estudando a possibilidade de criar uma binding library para iOS/Android.

**Comment by balivo** [comment]
Bom... Vou testar isso esse fds @filpgame ... rs... QQ coisa mando meu PR pra vocês...
Estive no TDC e falei com o pessoal lá do stand, deixei meu contato, estou precisando tanto disso que poderia ajudar vocês a fazer se fosse o caso... rs...

**Comment by programadorthi** [comment]
@filpgame Notei que vocês tem uma API para o POS SDK. Não seria melhor adaptar essa API utilizando .Net Standard? Acho que a parte de Binding do Xamarin não é uma boa alternativa.

Utilizando a API do POS SDK vocês só teriam que criar uma biblioteca que trabalha com requisições HTTP e liberar via Nuget.

Bom, é uma sugestão que acho que seria mais prático e rápido para o @balivo .

Obs.: Não sei se é possível, pois imagino que cada plataforma tem suas peculiaridades.

**Comment by balivo** [comment]
Então @programadorthi seria perfeito se fosse feito dessa forma, escrever a SDK para Xamarin.Android e Xamarin.Forms.

Mas como não tem previsão, não tem muito o que fazer...

[]'s

**Comment by balivo** [comment]
@filpgame na SDK de vocês, é fornecida uma interface para digitação dos dados de cartão?

Por exemplo: Usuário vai realizar uma transação via cartão de crédito, desenvolvedor passa para a SDK os parâmetros da transação (Valor, permiti parcelamento ou não, etc) e a SDK "cuida" de captar os dados do cartão do usuário abrindo um ambiente seguro para tal e finaliza a transação, retornando para a aplicação "hospedeira".

**Comment by bragattokk** [comment]
@balivo 
Sim, o processo acontece exatamente como você mencionou, na aplicação de integração com a SDK, deve ser feita toda a configuração de chamada dos dados da compra (valor, opção de captura e etc.) e a comunicação final com o SDK no Provider Transaction para enviar esses dados e dar continuidade ao processo de captura dos dados do cartão (via pinpad) de maneira segura e envio dessas informações ao emissor final.
